### DISCUSSION TITLE
New Exploit found unblock chrome://flags and chrome://policy and many more chrome urls

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1278] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
also ran on boby vm , instructions for that here https://github.com/Blobby-Boi/BlobeVM?tab=readme-ov-file
could also work on a home PC.
video steps :

https://github.com/user-attachments/assets/39dea3c6-98d1-4023-a72a-4cf74a7046a8

**Comment by jeffrey6576i** [comment]
this is cool but sadly its blocked.

**Reply by ghost** [reply]
blocked by what extension?? you could also do it on a normal computer with chrome no emulator needed.
plus somebody could host the bobby vm  on a website  or you could , there are many turtuals

**Reply by jeffrey6576i** [reply]
well no i didn't mean blocked it just doesn't work for me lol

**Reply by ghost** [reply]
wydm

**Reply by ghost** [reply]
what happen

**Reply by jeffrey6576i** [reply]
i logged in with my school account and it doesn't do anything like i dont know what to do i watched the video several times and i did every single move you did but no work

**Reply by lolpc100** [reply]
check https://github.com/coder0-code/BlobeVM

**Reply by ghost** [reply]
Oh sorry it didn’t work

**Comment by FunnyCorgi** [comment]
This is NOT flags or other urls for your chromebook. This is running on wherever you are running the VM, this is not a real exploit.

**Reply by jeffrey6576i** [reply]
fr? hm

**Reply by ghost** [reply]
wdym when I do it It shows me my school polices normally chrome://policy are blocked by the school enrollmen t

**Reply by HelloIloveGimkit2** [reply]
bro it's not an exploit to be nice it's just a small little cool thing you can do with blobevm
mean version (choose which one you want): THIS IS NOT A FUCKING EXPLOIT IT'S LEGIT NOTHING THIS DOES NOT HELP ANYONE THAT MUCH SO SHUT THE FUCK UP

**Reply by ghost** [reply]
sorry :( 😓

**Reply by ghost** [reply]
but Now  I view my policy's and do the extension Allowlist opener and the net://export policy password exploit

**Reply by HelloIloveGimkit2** [reply]
it's not really an exploit just a cool method to view ur policy and get the school wifi using another exploit

**Reply by jeffrey6576i** [reply]
calm down n**** damn

**Reply by HelloIloveGimkit2** [reply]
no n****

**Reply by jeffrey6576i** [reply]
do you have anything better? if yes THEN GIVE IT TO ME GIVE IT TO ME

**Reply by HelloIloveGimkit2** [reply]
yes
https://github.com/3kh0/ext-remover/discussions/1287

**Reply by HelloIloveGimkit2** [reply]
so fuck this exploit mine is a real one

**Reply by jeffrey6576i** [reply]
> so fuck this exploit mine is a real one

bet

**Reply by SlitheringLizard1** [reply]
gimkit your exploit is trash stop bragging

**Comment by Charlieis1nsan3** [comment]
sry bud but this isn't an exploit this is just blobevm
